src/models/models.py

```diff
@@ -1,26 +0,0 @@
-from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-import os
-
-app = Flask(__name__)
-
-app.config["SQLALCHEMY_DATABASE_URI"] = (
-    f"mysql+pymysql://{os.getenv('MYSQL_USER', 'user')}:"
-    f"{os.getenv('MYSQL_PASSWORD', 'password')}@"
-    f"{os.getenv('MYSQL_HOST', 'db')}/"
-    f"{os.getenv('MYSQL_DATABASE', 'mydatabase')}"
-)
-
-app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
-db = SQLAlchemy(app)
```

Comment:
@iiscas and @PedroATCosta, I think here we were creating a new flask app and not using the one from `app.py`.